skactiveml/visualization/_data_set.py

```diff
@@ -0,0 +1,116 @@
+import numpy as np
```

Comment:
maybe those functions are too specific and can be directly implemented within the notebook